src/Poseidon/CLI/Forge.hs

```diff
@@ -134,22 +135,22 @@ runForge (
     when (null relevantPackages) $ liftIO $ throwIO PoseidonEmptyForgeException
 
     -- get all individuals from the relevant packages
-    let allInds = getJointIndividualInfo relevantPackages
+    let allInds = getJointIndividualInfo relevantPackages []
 
     -- set entities to only packages, if --packagewise is set
     let relevantEntities =
             if packageWise
             then map (Include . Pac . makePacNameAndVersion) relevantPackages
             else entities
 
-    -- determine indizes of relevant individuals and resolve duplicates
-    let (unresolvedDuplicatedInds, relevantIndices) = resolveEntityIndices relevantEntities allInds
-
+    -- determine indizes of relevant individuals
+    let relevantIndices = resolveEntityIndices relevantEntities allInds
+        duplicateReport = reportDuplicateIndividuals . map (allInds !!) $ relevantIndices
     -- check if there still are duplicates and if yes, then stop
-    unless (null unresolvedDuplicatedInds) $ do
+    unless (null duplicateReport) $ do
         logError "There are duplicated individuals, but forge does not allow that"
         logError "Please specify in your --forgeString or --forgeFile:"
-        mapM_ (\(_,i@(IndividualInfo n _ _),_) -> logError $ show (SimpleInd n) ++ " -> " ++ show (SpecificInd i)) $ concat unresolvedDuplicatedInds
+        mapM_ (\(IndividualInfo n _ _ _ _, sugg) -> logError $ show (Ind n) ++ " -> " ++ show sugg) duplicateReport
```

Comment:
OK, good point. I will add a test.

Reply:
Hmm, this isn't trivial, because it's log output, which we currently suppress in `testLog`. I guess I need to figure out how to funnel stderr into a file and then add that to the GoldenTests. But I guess that would be a good thing to have.

Reply:
I'm aware - yes. That's why I rather meant: "Did you run some tests on the command line?". Would be excellent to have this properly tested, but maybe it's not worth the effort.

Reply:
OK, so indeed I think getting the error output into the golden tests is possible but a bit of an overkill for now. I just ran a manual test, and it looks OK:

```
stephan_schiffels@biomac04 ancient % trident forge -f "*Lamnidis_2018-1.0.0*, *Lamnidis_2018-1.0.1*" -d . -o ~/Desktop/test
trident v1.3.0.4 for poseidon v2.5.0, v2.6.0, v2.7.0, v2.7.1
https://poseidon-framework.github.io

[Info]    Checking base directories... 
[Info]    Searching POSEIDON.yml files... 
[Info]    5 found
[Info]    Checking Poseidon versions... 
[Info]    Initializing packages... 
[Warning] Duplicate individuals in package collection (XXX011, XXX012, XXX013, ...)
[Info]    Packages loaded: 5
[Info]    Unpackaged genotype data files loaded: 0
[Info]    Forging with the following entity-list: *Lamnidis_2018-1.0.0*, *Lamnidis_2018-1.0.1*
[Info]    2 packages contain data for this forging operation
[Error]   There are duplicated individuals, but forge does not allow that
[Error]   Please specify in your --forgeString or --forgeFile:
[Error]   <XXX011> -> <Lamnidis_2018-1.0.0:POP1:XXX011>
[Error]   <XXX011> -> <Lamnidis_2018-1.0.1:POP1:XXX011>
[Error]   <XXX012> -> <Lamnidis_2018-1.0.0:POP2:XXX012>
[Error]   <XXX012> -> <Lamnidis_2018-1.0.1:POP2:XXX012>
[Error]   <XXX013> -> <Lamnidis_2018-1.0.0:POP1:XXX013>
[Error]   <XXX013> -> <Lamnidis_2018-1.0.1:POP1:XXX013>
[Error]   <XXX014> -> <Lamnidis_2018-1.0.0:POP2:XXX014>
[Error]   <XXX014> -> <Lamnidis_2018-1.0.1:POP2:XXX014>
[Error]   <XXX015> -> <Lamnidis_2018-1.0.0:POP2:XXX015>
[Error]   <XXX015> -> <Lamnidis_2018-1.0.1:POP2:XXX015>
[Error]   <XXX016> -> <Lamnidis_2018-1.0.0:POP2:XXX016>
[Error]   <XXX016> -> <Lamnidis_2018-1.0.1:POP2:XXX016>
[Error]   <XXX017> -> <Lamnidis_2018-1.0.0:POP1:XXX017>
[Error]   <XXX017> -> <Lamnidis_2018-1.0.1:POP1:XXX017>
[Error]   <XXX018> -> <Lamnidis_2018-1.0.0:POP3:XXX018>
[Error]   <XXX018> -> <Lamnidis_2018-1.0.1:POP3:XXX018>
[Error]   <XXX019> -> <Lamnidis_2018-1.0.0:POP1:XXX019>
[Error]   <XXX019> -> <Lamnidis_2018-1.0.1:POP1:XXX019>
[Error]   Error in the forge selection: Unresolved duplicated individuals
```

Of course, one could try to be even smarter and kind of group per-individual. I'll take a quick look whether that makes sense.

Reply:
OK, I have changed this now to the following:

```
[Error]   There are duplicated individuals, but forge does not allow that
[Error]   Please specify in your --forgeString or --forgeFile:
[Error]   Duplicate individual <XXX011> (please specify)
[Error]     <XXX011> -> <Lamnidis_2018-1.0.0:POP1:XXX011>
[Error]     <XXX011> -> <Lamnidis_2018-1.0.1:POP1:XXX011>
[Error]   Duplicate individual <XXX012> (please specify)
[Error]     <XXX012> -> <Lamnidis_2018-1.0.0:POP2:XXX012>
[Error]     <XXX012> -> <Lamnidis_2018-1.0.1:POP2:XXX012>
[Error]   Duplicate individual <XXX013> (please specify)
[Error]     <XXX013> -> <Lamnidis_2018-1.0.0:POP1:XXX013>
[Error]     <XXX013> -> <Lamnidis_2018-1.0.1:POP1:XXX013>
[Error]   Duplicate individual <XXX014> (please specify)
[Error]     <XXX014> -> <Lamnidis_2018-1.0.0:POP2:XXX014>
[Error]     <XXX014> -> <Lamnidis_2018-1.0.1:POP2:XXX014>
[Error]   Duplicate individual <XXX015> (please specify)
[Error]     <XXX015> -> <Lamnidis_2018-1.0.0:POP2:XXX015>
[Error]     <XXX015> -> <Lamnidis_2018-1.0.1:POP2:XXX015>
[Error]   Duplicate individual <XXX016> (please specify)
[Error]     <XXX016> -> <Lamnidis_2018-1.0.0:POP2:XXX016>
[Error]     <XXX016> -> <Lamnidis_2018-1.0.1:POP2:XXX016>
[Error]   Duplicate individual <XXX017> (please specify)
[Error]     <XXX017> -> <Lamnidis_2018-1.0.0:POP1:XXX017>
[Error]     <XXX017> -> <Lamnidis_2018-1.0.1:POP1:XXX017>
[Error]   Duplicate individual <XXX018> (please specify)
[Error]     <XXX018> -> <Lamnidis_2018-1.0.0:POP3:XXX018>
[Error]     <XXX018> -> <Lamnidis_2018-1.0.1:POP3:XXX018>
[Error]   Duplicate individual <XXX019> (please specify)
[Error]     <XXX019> -> <Lamnidis_2018-1.0.0:POP1:XXX019>
[Error]     <XXX019> -> <Lamnidis_2018-1.0.1:POP1:XXX019>
[Error]   Error in the forge selection: Unresolved duplicated individuals
```

... more verbose, but arguably more helpful!

Reply:
Beautiful :+1: